lib/local_storage_service.dart

```diff
@@ -96,4 +97,22 @@ class LocalStorageService {
     final result = await prefs.setString(_usersKey, usersMapJsonEncoded);
     return result;
   }
+
+  Future<void> handleExit() async {
```

Comment:
i do not see point of this method, lets just use deleteLastEmail outside instead of handleExit